.wordpress-org/logo.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Would WP be correctly detecting this file. I see that `icon.svg` is the name used by the [User Switching](https://github.com/johnbillion/user-switching/blob/bb97547eb9afea484550bc7bc34a670d9d08d528/.wordpress-org/icon.svg) & [Web Stories](https://github.com/google/web-stories-wp/blob/46b5b7672a795769f64d590711f3c50de89c221f/.wordpress-org/icon.svg) plugins, for example.

Reply:
Oops. Yeah, I neglected to rename it.